### DISCUSSION TITLE
Floppy disk "stack" to allow quick disk switching

### DISCUSSION THREAD

**Discussion by Zack13358** [86Box/86Box#3472] [Ideas]
The only emulators I can think of that have this feature is DOSbox and WebMSX, where you can select multiple floppies (or even a folder) in a "stack" and then press a hotkey to quickly go to the next floppy in the stack. 

This would be useful for programs that have a lot of floppies like win95, win98, MS office 4.x, etc.

**Comment by stockerta** [comment]
Yes, something like a next disk button that iterates over image files in a folder.
For example if I open a disk image named 'disk01.img' it would then insert 'disk02.img'
